authors/apps/articles/views.py

```diff
@@ -156,6 +156,34 @@ def post(self, request, slug):
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
 
+class CommentChoiceView(ListCreateAPIView):
+    """Implements the like and dislike endpoints."""
+    serializer_class = CommentSerializer
+    model = Comment
+    vote_type = None
+    manager = None
+
+    def post(self, request, slug, pk):
```

Comment:
@NLSanyu, How do you see if we reduced this duplication? Don't you think it will affect our maintainability will decline to a B? What about we create a superclass where this CommentView and ArticleView can inherit from?

Reply:
@akram256 I am working on it